guides/doc-Managing_Hosts/topics/Global_Registration.adoc

```diff
@@ -0,0 +1,138 @@
+[id="Global_Registration"]
+= Registering Host with Global Registration
+
+With Global Registration feature hosts can be registered to the Foreman with
+just one command.
+Main power of this feature came from utilization of provisioning templates,
+which gives user complete control over the process of the host registration.
+
+
+.Prerequisites
+- Foreman user must have `create_hosts` permission
+- Host OS must have assigned Registration template.
+  Go to the template detail > Associations & select the OS
+  Then on OS detail (Hosts > Operating Systems) set default registration template
+- Root privileges on the host that's going to be registered.
+
+
+.Procedure
+. Login to {ProjectName}
+. Go to Hosts > All Hosts > Register Host
+. Set required parameters and generate the command
+. Log in to the Host
+. Run command generated in {ProjectServer}
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# Example:
+curl http://_{foreman-example-com}_/register | bash
+----
++
+
+
+.Templates & snippets
+
+[cols=2*]
+|===
+|Global Registration Template
+|Contains steps for registration of host to the Foreman. Rendered when accessing `/register` endpoint.
+
+|Host Registration Template
+|In Foreman named `Linux registration default`, contains commands for Host setup.
+
+|`remote_execution_ssh_keys` snippet
+| Used in Host registration template, deploy SSH keys to the host. Only when `host_registration_remote_execution` parameter is `true`. Part of Host registration template
+
+|`insights` snippet
+| Download & install insights client. Only for Red Hat OS & when global parameter `host_registration_insights` is set to true. Part of Host registration template
+|===
+
+
+.Registration parameters
+* Organization
+* Location
+* Host Group
+* Operating System - No need to specify for registration with subscription-manager
+* Proxy
+* Setup Insights - Install insights client & register host. Override value of `host_registration_insights` global parameter on the Host level. Plus only for Red Hat OS.
+* Remote Execution - Override value of `host_registration_remote_execution` global parameter on the Host level.
+* Token lifetime
+
+
+.Settings
+* For changing default Global Registration template go to Administer > Settings > Provisioning
```

Comment:
It would be good to mention why would one want to change that. E.g. if you want to customize the default registration behavior, instead of unlocking and modifying the default template, you can clone it and change it via settings. The default template gets overriden on every Foreman upgrade so custom changes would be lost. (please change the text as needed)